.azure/wheels.yml

```diff
@@ -27,6 +27,7 @@ jobs:
           python -m pip install --upgrade pip
           pip install cibuildwheel==2.11.2
           pip install -U twine
+          rustup override set stable
```

Comment:
I realize I'm too late now that this has already merged, but I don't think this will actually have the desired effect on linux (it should work fine on macOS and windows though). On linux the builds are all done in docker images (with compatible versions of glibc and other system libs) and setting the override via rustup outside the cibuildwheel process won't actually effect the build process because the override gets set to a user config file which won't be inside the docker container. The easiest way to probably do this is to just set the env variable to override the toolchain in the cibuildwheel config here: https://github.com/Qiskit/qiskit-terra/blob/main/pyproject.toml#L23

Reply:
I pushed up a PR to fix this here: https://github.com/Qiskit/qiskit-terra/pull/9596